src/Commands/NotificationCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Soyhuce\Somake\Commands;
+
+use Illuminate\Console\Command;
+use Soyhuce\Somake\Commands\Concerns\AsksDomain;
+use Soyhuce\Somake\Support\Finder;
+use Soyhuce\Somake\Support\Writer;
+use function Laravel\Prompts\outro;
+use function Laravel\Prompts\text;
+
+class NotificationCommand extends Command
+{
+    use AsksDomain;
+
+    /** @var string */
+    public $signature = 'somake:notification';
+
+    /** @var string */
+    public $description = 'Generates a Notification in Domain';
+
+    public function handle(Finder $finder, Writer $writer): void
+    {
+        $notification = text(label: 'What is the Notification name ?', required: true);
+
+        $queued = $this->confirm(question: 'Should the notification be queued ?');
+
+        $multilingual = $this->confirm(question: 'Should the notification be multilingual ?', default: true);
+
+        $domain = $this->askDomain($finder->domains());
+
+        $writer
+            ->write(
+                'notification',
+                [
+                    'notification' => $notification,
+                    'queued' => $queued,
+                    'multilingual' => $multilingual,
+                ]
+            )
+            ->toPath($finder->domainPath("{$domain}/Notifications/{$notification}.php"));
+
+        $notificationFqcn = "Domain\\{$domain}\\Notifications\\{$notification}";
+
+        outro("The {$notificationFqcn} class was successfully created !");
+    }
```

Comment:
Please ask if the developer want a unit test to be generated. A pre-filled unit test specific to the notification should be great too

Reply:
Done in 33ae756703a86b7e53e299633896b7d8168f972f